source/positionctrl.cpp

```diff
@@ -66,7 +68,8 @@ bool PositionCtrl::Enable(bool enable)
 void PositionCtrl::OnClipboardText(wxClipboardTextEvent& evt)
 {
 	Position position;
-	if(posFromClipboard(position.x, position.y, position.z)) {
+	Editor* editor = g_gui.GetCurrentEditor();
```

Comment:
This is another situation where g_gui's dependency is not good. PositionCtrl already have the maxx and maxy properties in contructor. You should use them

Reply:
Should be addressed with this commit https://github.com/hjnilsson/rme/pull/268/commits/b98c4f4c7e05ece1b177249275d8b4d7ce6690dc